modules/3d-tiles/src/lib/tileset/tileset-3d.js

```diff
@@ -1,19 +1,15 @@
 // This file is derived from the Cesium code base under Apache 2 license
 // See LICENSE.md and https://github.com/AnalyticalGraphicsInc/cesium/blob/master/LICENSE.md
 
-import {Matrix4, Vector3} from 'math.gl';
+import {Matrix4, Vector3} from '@math.gl/core';
 import {Ellipsoid} from '@math.gl/geospatial';
-import {Stats} from 'probe.gl';
+import {Stats} from '@probe.gl/stats';
```

Comment:
Aren't you moving `Tileset3D` to the `tiles` module per your RFC? Or that is "phase 2"?

Reply:
yes, it will be in the following PR